web/static/css/app.css

```diff
@@ -38,3 +38,19 @@ li:last-child{float: right;}
 div {
   margin-top: 70px;
 }
+button {
+  border: 10px solid #2c3e50;
```

Comment:
let's specify an id on this button so this doesn't style every button on the site